weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx

```diff
@@ -46,15 +46,26 @@ export const ObjectViewer = ({apiRef, data, isExpanded}: ObjectViewerProps) => {
       opGet({uri: constString(ref)} as any)
     );
     const refQueries = refValueNodes.map(node => client.query(node));
-    Promise.all(refQueries).then(values => {
+    Promise.allSettled(refQueries).then(values => {
       const refValues: RefValues = {};
       for (const [r, v] of _.zip(refs, values)) {
-        if (!r) {
+        if (!r || !v) {
```

Comment:
Should probably remove the comment on line below.

Reply:
think it is still valid (b/c we expect the same length results

Reply:
You're right, nevermind.